tools/image_processing/imagej2/imagej2_watershed_binary/imagej2_watershed_binary.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<tool id="imagej2_watershed_binary" name="Watershed segmentation" version="1.0.0">
+    <description>of binary image with ImageJ2</description>
+    <macros>
+        <import>imagej2_macros.xml</import>
+    </macros>
+    <expand macro="fiji_20141125_requirements" />
```

Comment:
Ok, I will change it to be just fiji_requirements.  If I remember correctly, I believe I named it this way since the Fiji site states that Fiji is still in beta, and things can change.  But clearly having to change all tools when the next version of Fiji is released is not optimal if changes don't affect tool results.